content/docs/capabilities/ppl.mdx

```diff
@@ -132,24 +136,22 @@ PPL supports many different criteria:
 | `claim` | Anything. Typically a string. | Returns true if a token claim matches the supplied value **exactly**. The claim to check is determined via the sub-path. <br/> For example, `claim/family_name: Smith` matches if the user's `family_name` claim is `Smith`. |
 | `client_certificate` | [Certificate matcher] | Returns true if a client presented a TLS certificate matching the provided condition. |
 | `cors_preflight` | Anything. Typically `true`. | Returns true if the incoming request uses the `OPTIONS` method and has both the `Access-Control-Request-Method` and `Origin` headers. Used to allow [CORS pre-flight requests]. |
+| \* `date` | [Date Matcher] | Returns true if the time of the request matches the constraints. |
+| \* `day_of_week` | [Day of Week Matcher] | Returns true if the day of the request matches the constraints. |
 | `device` | [Device matcher] | Returns true if the incoming request includes a valid device ID or type. |
 | `domain` | [String Matcher] | Returns true if the logged-in user's email address domain (the part after `@`) matches the given value. |
 | `email` | [String Matcher] | Returns true if the logged-in user's email address matches the given value. |
+| \* `groups` | [String Matcher] | Returns true if a user's group ID matches the supplied value **exactly**. `groups` data is only available after a successful directory sync. See [Identity Providers](/docs/identity-providers) for vendor-specific directory sync steps. |
```

Comment:
I believe `groups` is not using String Matcher, but rather `String List Matcher` that has a single operator `has` that checks that a given string is present in a given list of strings.